### DISCUSSION TITLE
Help compilation

### DISCUSSION THREAD

**Discussion by zdzicho13** [HASwitchPlate/openHASP#260] [General]
Hello
I have an ESP32 WROVER module with a 2.4 inch ILI9341 where there is a connection:
      TFT_DC = 0
      TFT_CS = 5
      TFT_RST = none
      TFT_BCKL = 23
      TOUCH_CS = 21
Can you have an openHASP compilation for me?

**Comment by fvanroie** [comment]
The easiest way is to edit (or copy) the [user_setups/esp32/d1-mini-esp32_ili9341.ini](https://github.com/HASwitchPlate/openHASP/blob/master/user_setups/esp32/d1-mini-esp32_ili9341.ini) file and compile that firmware either [locally](https://haswitchplate.github.io/openHASP-docs/latest/compiling/local/) or on [gitpod](https://haswitchplate.github.io/openHASP-docs/latest/compiling/gitpod/).
